packages/eslint-plugin/README.md

```diff
@@ -75,22 +79,33 @@ The name must be equal to the name of a variable or a property an entity is assi
 
 ```ts
 const count = atom(0, 'count')
+```
+
+When atom is assigned to a property of an object assigned to a variable, variable name must be specified before a period (`.`):
 
-const someNamespace = {
-  count: atom(0, 'count'),
+```ts
+const atomsRec = {
+  count: atom(0, 'atomsRec.count'),
 }
 ```
 
-When creating atoms dynamically with factories, you can also specify the "namespace" of the name before the `.` symbol:
+When creating units within `reatom*`-named factory functions, you can also specify the "namespaces" of unit names before period. The fragment before period is called the domain. Domain value must be equal to the name of the factory function excluding the `reatom` prefix:
 
 ```ts
-const reatomFood = (config: {
-  name: string
-  calories: number
-  fat: number
-  carbs: number
-  protein: number
-}) => {
+const reatomFood = (config: { name: string; calories: number; fat: number; carbs: number; protein: number }) => {
+  const { name } = config.name
```

Comment:
Why do we need a name there? It seems like it is unused, and we have an example where it is used starting on line 108.